builder/dockerignore/dockerignore.go

```diff
@@ -21,7 +21,8 @@ func ReadAll(reader io.ReadCloser) ([]string, error) {
 
 	for scanner.Scan() {
 		pattern := strings.TrimSpace(scanner.Text())
-		if pattern == "" {
+		// Blank lines or lines starting with # are ignored.
+		if pattern == "" || strings.HasPrefix(pattern, "#") {
```

Comment:
Thanks @thaJeztah @duglin I updated the pull request. Now only lines starting with `#` (at position 1) are considered as comments.